srcopsmetrics/cli.py

```diff
@@ -160,9 +161,12 @@ def cli(
     for project in repos:
         os.environ["PROJECT"] = project
 
+    today = date.today()
+    yesterday = today.replace(day=today.day - 1)
```

Comment:
This is going to give error, if you don't change also month and eventually year. 
For example today is `2021-06-01`, if you run `yesterday = today.replace(day=today.day - 1)` you will get error: `ValueError: day is out of range for month`, unless you change also month.

A more robust/easier solution is to use:
```
from datetime import date, timedelta
today = date.today()
yesterday = today - timedelta(days=1)
```
which returns correctly `2021-05-31`